bread/contrib/triggers/models.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+
+import htmlgenerator as hg
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.contrib.contenttypes.models import ContentType
+from django.core.mail import send_mail
+from django.db import models
+from django.template import engines
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from bread.formatters import is_email_simple
+from bread.querysetfield import QuerysetField
+
+syntax_email_field = """
+Syntax:
+- Email: example@example.com
+- User/Group: @username
+- From object: manager.email (actually object.manager.email)
+
+Multiple values van be separated by comma , e.g.
+boss@example.com, @adminuser, @reviewteam, primary_email_address.email
+"""
+
+syntax_event_field = """
+Database change: added, changed, deleted
+Date or time in ISO 8601 format: YYYY-MM-DD hh:mm:ss
+Date field from object: field:reminder.date 2 weeks ago
+
+Multiple values van be separated by comma , e.g.
+changed, @reminder.date in 2 weeks
+"""
+
+
+class Action(models.Model):
+    description = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.description
+
+    def run(self, object):
+        raise NotImplementedError()
+
+    class Meta:
+        verbose_name = _("Action")
+        verbose_name_plural = _("Actions")
+
+
+class SendEmail(Action):
+    email = models.CharField(_("Email"), max_length=255, help_text=syntax_email_field)
+    subject = models.CharField(
+        _("Subject"),
+        max_length=255,
+        help_text=_(
+            "Will be rendered as a Django template with the name 'object' in the context"
+        ),
+    )
+    message = models.TextField(
+        _("Message"),
+        help_text=_(
+            "Will be rendered as a Django template with the name 'object' in the context"
+        ),
+    )
+
+    def run(self, object):
+        recepients = []
```

Comment:
typo: I think `recepients` should be `recipients`

Reply:
Thanks, fixed.